modules/core/server/views/partials/footer.server.view.html

```diff
@@ -2,31 +2,47 @@
         role="contentinfo"
         class="container hidden-print hidden-xs"
         ng-if="!app.isFooterHidden">
-  <div class="row">
-    <div class="col-xs-12" itemscope itemtype="http://schema.org/Organization">
-      <a ui-sref="foundation" itemprop="name">
-        Trustroots Foundation
-      </a>
-      <link itemprop="url" href="https://www.trustroots.org/">
-      <link itemprop="logo" href="https://www.trustroots.org/img/logo/color.png">
-
-      <ul class="list-inline text-center">
+  <hr>
+  <div class="navbar">
+    <div class="col-xs-6">
+      <ul class="navbar">
+        <li itemscope itemtype="http://schema.org/Organization">
+          <a ui-sref="foundation" itemprop="name">
+            Trustroots Foundation
+          </a>
+          <link itemprop="url" href="https://www.trustroots.org/">
+          <link itemprop="logo" href="https://www.trustroots.org/img/logo/color.png">
```

Comment:
We can render `https://www.trustroots.org` dynamically with `{{hostPort}}` 

See examples from `modules/core/server/views/layout.server.view.html`